### PR TITLE
New function pgr betweenness centrality

### PR DESCRIPTION
**Pull request by bedupako12mas**
This PR adds one new pgRouting function:
- ``pgr_betweennessCentrality``: Returns the relative betweenness centrality of all vertices of a graph.

This is a final merge PR containing commits from several different PRs combined.

The intermediate PRs made on the [GSoC-pgRouting](https://github.com/pgRouting/GSoC-pgRouting/) repository are as follows:

| Description |  Pull Requests |
| -----------| --------------------- |
| pgr_betweennessCentrality | <ul><li>pgRouting/GSoC-pgRouting#378</li><li>pgRouting/GSoC-pgRouting#383</li><li>pgRouting/GSoC-pgRouting#386</li><li>pgRouting/GSoC-pgRouting#388</li><li>pgRouting/GSoC-pgRouting#390</li><li>pgRouting/GSoC-pgRouting#392</li><li>pgRouting/GSoC-pgRouting#393</li><li>pgRouting/GSoC-pgRouting#396</li><li>pgRouting/GSoC-pgRouting#398</li><li>pgRouting/GSoC-pgRouting#399</li><li>pgRouting/GSoC-pgRouting#402</li><li>pgRouting/GSoC-pgRouting#403</li></ul> |

### CODE REVIEW

**Comment by cvvergara** [comment]
Please wait until all the mentors approve to rebase.
**Do not merge, do a rebase.**
Add the link to this PR to your final report.

**Review by krashish8 — APPROVED** [review]
Thanks @bedupako12mas for your contributions! 🙌 
Great work! 🚀

**Review by cvvergara — APPROVED** [review]

**Review by robe2 — APPROVED** [review]

**Pull request merged**
